assets/package.json

```diff
@@ -53,19 +53,17 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^4.13.1",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "mini-css-extract-plugin": "^2.8.1",
+    "sass": "^1.69.5",
+    "sass-loader": "^13.3.2",
```

Comment:
Some of these appear to have escaped from their formerly-sorted order (S here is followed by P). Can we do a quick re-sort?